components/dexidp/values-generic.yaml

```
@@ -27,10 +27,12 @@ config:
       config:
         # Required, without v3 suffix.
         keystoneHost: http://keystone-api.openstack.svc.cluster.local:5000
-        # Required, admin user credentials to connect to keystone.
+        # domain ID where users will authenticate against
         domain: default
```

small suggestion to make it more obvious that we expect ID, not a name here:
```suggestion
        domain: 0123456789abcd
```